pkg_building.es.Rmd

```diff
@@ -1,513 +1,511 @@
+Para más explicaciones sobre las licencias, consulta el [Libro de paquetes de R](https://r-pkgs.org/license.html).
 
-## Testing
+## Probando
 
-* All packages should pass `R CMD check`/`devtools::check()` on all major platforms.
+- Todos los paquetes deben pasar `R CMD check`/`devtools::check()` en las principales plataformas.
 
-* All packages should have a test suite that covers major functionality of the package. The tests should also cover the behavior of the package in case of errors.
+- Todos los paquetes deben tener un conjunto de pruebas que cubran la funcionalidad principal del paquete. Las pruebas deben cubrir también el comportamiento del paquete en caso de error.
 
-* It is good practice to write unit tests for all functions, and all package code in general, ensuring key functionality is covered. Test coverage below 75% will likely require additional tests or explanation before being sent for review.
+- Es una buena práctica escribir pruebas unitarias para todas las funciones, y para todo el código del paquete en general, asegurando que se cubra la funcionalidad clave. La cobertura de las pruebas por debajo del 75% probablemente requerirá pruebas adicionales o una explicación antes de ser enviada para su revisión.
 
-* We recommend using [testthat](https://testthat.r-lib.org/) for writing tests. Strive to write tests as you write each new function. This serves the obvious need to have proper testing for the package, but allows you to think about various ways in which a function can fail, and to _defensively_ code against those. [More information](https://r-pkgs.org/tests.html).
+- Recomendamos utilizar [testthat](https://testthat.r-lib.org/) para escribir las pruebas. Esfuérzate por escribir pruebas a medida que escribes cada nueva función. Esto responde a la necesidad obvia de tener pruebas adecuadas para el paquete, pero te permite pensar en varias formas en las que una función puede fallar, y en *defensivamente* defensivamente contra ellas. [Más información](https://r-pkgs.org/tests.html).
 
-* Tests should be easy to understand. We suggest reading the blog post [_"Why Good Developers Write Bad Unit Tests"_](https://mtlynch.io/good-developers-bad-tests/) by Michael Lynch.
+- Las pruebas deben ser fáciles de entender. Te sugerimos que leas la entrada del blog [*"Por qué los buenos desarrolladores escriben malas pruebas unitarias"*](https://mtlynch.io/good-developers-bad-tests/) de Michael Lynch.
 
-* Packages with Shiny apps should use a unit-testing framework such as [`shinytest`](https://rstudio.github.io/shinytest/articles/shinytest.html) to test that interactive interfaces behave as expected.
+- Los paquetes con aplicaciones Shiny deberían utilizar un marco de pruebas unitarias como [`shinytest`](https://rstudio.github.io/shinytest/articles/shinytest.html) para comprobar que las interfaces interactivas se comportan como se espera.
```

Comment:
```suggestion
- Los paquetes con aplicaciones Shiny deberían utilizar un marco de pruebas unitarias como [`shinytest2`](https://rstudio.github.io/shinytest2/) o [`shinytest`](https://rstudio.github.io/shinytest/articles/shinytest.html) para comprobar que las interfaces interactivas se comportan como se espera.
```

@paocorrales this is something I added to the English verstion

Reply:
I'm copying and pasting this to the Spanish version. We'll see how this goes when we merge the PRs!

Reply:
thank you and sorry!